servers/visual/visual_server_raster.h

```diff
@@ -1015,6 +1015,9 @@ class VisualServerRaster : public VisualServer {
 	BIND3(instance_set_surface_material, RID, int, RID)
 	BIND2(instance_set_visible, RID, bool)
 
+	BIND2(instance_set_custom_aabb, RID, Rect3)
```

Comment:
I would just use a single function, and assume AABB() (empty bounding box) is disabled.

Reply:
Also needs a rebase since the re-rename from Rect3 to AABB.